src/createServer.js

```diff
@@ -1,3 +1,39 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { findErrors } = require('./validation');
+
+const createServer = () => {
+  return http.createServer((req, res) => {
+    res.setHeader('Content-type', 'application/json');
+
+    const { url } = req;
+    const normURL = new URL(url, 'http://localhost:5700');
+    const [text] = url.slice(1).split('?');
+    const caseName = normURL.searchParams.get('toCase');
+
+    const errorsList = findErrors(text, caseName);
+
+    if (errorsList.length > 0) {
+      res.statusCode = 400;
+
```

Comment:
`res.statusMessage = 'Bad request';`

From README: "If something is not correct, you should respond with 400 status, `Bad request` statusText and the payload."